setup.py

```diff
@@ -134,7 +134,10 @@ def check_dependency_versions(min_versions):
                               (spversion, min_versions['scipy']))
 
     try:
-        from pandas.version import short_version as pversion
+        import pandas
+        #FIXME: this will break for pandas 1.0.0.  Needs elaborate parsing now,
+        # due to pandas removing version.short_version
```

Comment:
The only alternative here that I can think of without adding a significant amount of code is to use ``NumpyVersion`` from ``statsmodels/compat/scipy.py``. But I don't think that's better (pandas version string format) may not remain compatible with numpy's. So better solve it properly with more code later.